src/platform/forms/components/review/ClientError.jsx

```diff
@@ -0,0 +1,54 @@
+// libs
+import React from 'react';
+
+// platform - forms components
+import Column from 'platform/forms/components/common/grid/Column';

```

Comment:
What do you think about putting an `index.js` in `common/grid/` so we can
```js
import { Column, Row } from 'platform/forms/components/common/grid';
```
?

Reply:
Love this idea and I think we will move more towards that pattern as we become a library. 😃